docs/src/man/doctests.md

```diff
@@ -389,3 +389,28 @@ Alternatively, you can also pass the `doctest = :fix` keyword to [`makedocs`](@r
 Doctesting can be disabled by setting the [`makedocs`](@ref) keyword `doctest = false`.
 This should only be done when initially laying out the structure of a package's
 documentation, after which it's encouraged to always run doctests when building docs.
+
+## Testing README.md
+
+The easiest way to also run doctests on the README.md file is to add the readme to the
+docs of the module:
+
+```julia
+module MyModule
+
+function _update_module_doc()
+    path = joinpath(pkgdir(MyModule), "README.md")
```

Comment:
```suggestion
    path = joinpath(@__DIR__, "..", "README.md")
```

I think it is better to use a relative path here rather than `pkgdir` (which is much more complicated).

Reply:
Nice one :+1: